### DISCUSSION TITLE
Forma de entregar el wireframe y el UI

### DISCUSSION THREAD

**Discussion by gustavolire23** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#137] [Q&A] [answered]
Hola compañeros, quería saber en qué formato entregaran el UI y el wireframe, será con el link del proyecto de figma o lo agregarán como imagen?
Gracias

**Comment by armap99** [comment]
Al momento de llenar el formulario puedes poner el enlace de tu proyecto de figma pero también te piden un enlace al repositorio de donde debes de poner los enlaces, yo te recomiendo que por cada enlace crees un archivo tipo .md y dentro el enlace a tus practicas o subir las imagen directamente a tu repositorio pdf a tu repositorio y poner el enlace de ese archivo localizado dentro de tu repo en el formulario.

**Reply by gustavolire23** [reply]
Gracias compañero.

**Reply by JuanSalvador73** [reply]
Gracias igual, ayuda mucho esa info.

**Comment by romarpla** [comment, marked as answer]
[Link de publicación de Teams](https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628)

Este es el link dentro del equipo de Teams para que puedan ver el link del Formulario

**Reply by gustavolire23** [reply]
Gracias mission commander.
